### PR TITLE
Very small stuff

### PR DESCRIPTION
**Pull request by viklund**
Some convenience stuff, very little updated code.

### CODE REVIEW

File: docker/Makefile [hunk 1]
```diff
@@ -24,6 +24,12 @@ network:
 up:network private/cega.yml private/lega.yml
 	@docker-compose -f private/cega.yml -f private/lega.yml up -d
 
+volumes:
+	docker volume create lega_db lega_inbox lega_s3
+
+clean-volumes:
+	docker volume rm lega_db lega_inbox lega_s3
```
Comment (silverdaz) [inline docker/Makefile]: Isn't there a `-y` or something similar to force it?
Or do you want the make to fail there, on purpose?
If you do, add a dash in front of the command on line 28 and 31.

File: docker/README.md [hunk 2]
```diff
@@ -22,6 +22,11 @@ These networks are reflected in their corresponding YML files
 
 The passwords are in `private/lega/.trace` and the errors (if any) are in `private/.err`.
 
+If you don't have the volumes already, you can create them with
+
+    make volumes
```
Comment (silverdaz) [inline docker/README.md]: Didn't you like that docker-compose would create the volumes ?
It doesn't harm to have that target in the makefile, and I'm just curious to why.
Is it anything else than conveniency ?

**Review by silverdaz — APPROVED** [review]
I approve like that, but look at the questions I added in the comments, please.

File: docker/README.md (hunk 2 above)
Comment (viklund) [reply docker/README.md]: On my machine docker-compose does not create the volumes. It just complains that it's not there.

File: docker/Makefile (hunk 1 above)
Comment (viklund) [reply docker/Makefile]: At least for the rm it is probably good if it triggers fail of the make process. As for create, I don't know.

File: docker/README.md (hunk 2 above)
Comment (viklund) [reply docker/README.md]: But now it does, that was strange. I'll remove that then. Now I'm confused about why I had to create them manually before...

**Pull request merged**